apps/web/components/booking/BookingListItem.tsx

```diff
@@ -378,8 +378,15 @@ function BookingListItem(booking: BookingItemProps) {
             )}
             {booking.attendees.length !== 0 && (
               <DisplayAttendees
-                attendees={booking.attendees}
-                user={booking.user}
+                attendees={[
+                  ...booking.attendees,
+                  ...booking.user.reduce((memberArray, userEntry) => {
+                    if (userEntry.email !== user?.email)
+                      memberArray.push({ name: userEntry.name || "Nameless", email: userEntry.email });
+                    return memberArray;
+                  }, [] as AttendeeProps[]),
+                ]}
+                user={booking.user.find((userEntry) => userEntry.email === user?.email) || booking.user[0]}
```

Comment:
On the booking page, set the current user as the first attendee and the rest as other attendees. This means the current user will not be repeated.